### ISSUE TITLE
Top and Footer navigation not showing all links on smaller screens.

### ISSUE THREAD

**Issue by keithmifsud** [learnk8s/learnk8s.io#55]
@danielepolencic Is this intentional? 

- [ ] On XS screen sizes the top navigation does not show.
- [ ] On SM screen sizes the top navigation only shows some links.

Same goes for the footer.

I think I should rebuild the navigation so that a "burger menu" is shown for XS and SM screens. I also think that I should rebuild the footer so that all content is shown in a responsive manner.

Please let me know your thoughts. I'm able to work on this task.

**Comment by keithmifsud** [comment]
@danielepolencic Please let me know your thoughts about this issue when you get a chance.

**Comment by danielepolencic** [comment]
The missing links are on purpose. I didn’t have much time to code the burger menu. Also the experience on mobile is different from desktop. If you land on a website with your mobile, you are probably on the go. You probably need something quick and don’t want to spend to much time reading all the content.

I didn’t spend much time thinking about what’s right/wrong or how we could deliver a decent experience for mobile users.

I agree that the experience could be improved but before we start coding, I’d like to spend some time thinking about what’s useful and what’s not.

If you have ideas, please share.

Anyway, I think this is not a priority atm. we should come back to this after we’ve sorted the newsletter.

**Comment by keithmifsud** [comment]
I think that at least a contact link should be shown at the top when viewing the site in xs screens. The visitor needs to scroll down to the footer to get any links. I don't agree that folks need to be on the go to view websites from their phone and even if so, it does not necessarily mean that they are in a rush. They can be sitting on a flight/train /taxi etc. for hours :)

@almoretti what the marketers view on this?

**Comment by danielepolencic** [comment]
We have a burger menu now.
